index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Richard Rabulan | Web Developer</title>
+    <link rel= "stylesheet" type="text/css" href= "normalize.css">
+    <link rel= "stylesheet" type="text/css" href= "style.css">
+</head>
+<a name="top"></a>
+<body>
+    <header class="navBar featureImage">
+        <div class="container">
```

Comment:
do you need this extra layer of div? could the container class go on the header instead?

Reply:
container div has a width of 90%, the background of featureImage is 100%